src/sdg/producten/migrations/0052_rename_toelichting_fields.py

```diff
@@ -0,0 +1,43 @@
+# Generated by Django 3.2.16 on 2023-04-06 13:30
+
+from django.db import migrations
+from django.db.models import Func, Value, Q
+
+
+class Replace(Func):
+    function = "REPLACE"
+
+
+def replace_toelichting_field_omdat_value_with_dot(apps, schema_editor):
+    LocalizedProduct = apps.get_model("producten", "localizedproduct")
+
+    LocalizedProduct.objects.filter(
+        Q(product_valt_onder_toelichting__icontains=" omdat...")
+        | Q(product_valt_onder_toelichting__icontains="omdat...")
+        | Q(product_aanwezig_toelichting__icontains=" omdat...")
+        | Q(product_aanwezig_toelichting__icontains="omdat...")
```

Comment:
You made a typo I think.

You need to match " omdat..." and " omdat ..." (the difference is in the 2nd space char).